<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tili</class>
 <widget class="QDialog" name="Tili">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushKirjauduUlos">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushKirjauduUlos{
background-color:rgba(242, 140, 40,0.9); color: rgba(0, 0, 0, 0.8); border-style: outset;  border-width: 2px;   border-color: rgba(255, 140, 40,1);
color: white;
}
QPushButton#pushKirjauduUlos:hover{
background-color:rgba(230, 120, 40,0.9); color: rgba(0, 0, 0, 0.8); 
}</string>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushValinta">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushValinta{
background-color:rgba(242, 140, 40,0.9); color: rgba(0, 0, 0, 0.8); border-style: outset;  border-width: 2px;   border-color: rgba(255, 140, 40,1);
color: white;
}
QPushButton#pushValinta:hover{
background-color:rgba(230, 120, 40,0.9); color: rgba(0, 0, 0, 0.8); 
}</string>
   </property>
   <property name="text">
    <string>Vaihda Tili</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushTalletusNosto">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushTalletusNosto{
background-color:rgba(242, 140, 40,0.9); color: rgba(0, 0, 0, 0.8); border-style: outset;  border-width: 2px;   border-color: rgba(255, 140, 40,1);
color: white;
}
QPushButton#pushTalletusNosto:hover{
background-color:rgba(230, 120, 40,0.9); color: rgba(0, 0, 0, 0.8); 
}</string>
   </property>
   <property name="text">
    <string>Talletus </string>
   </property>
  </widget>
  <widget class="QPushButton" name="Push_Nosto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#Push_Nosto{
background-color:rgba(242, 140, 40,0.9); color: rgba(0, 0, 0, 0.8); border-style: outset;  border-width: 2px;   border-color: rgba(255, 140, 40,1);
color: white;
}
QPushButton#Push_Nosto:hover{
background-color:rgba(230, 120, 40,0.9); color: rgba(0, 0, 0, 0.8); 
}</string>
   </property>
   <property name="text">
    <string>Nosto</string>
   </property>
  </widget>
  <widget class="QLabel" name="Huomautus">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(240, 0, 0, 90); color:white;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>K채ytt채j채ll채 ainoastaan yksi tili</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lable_TiliNumero">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>70</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(242, 140, 40,0.8);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lable_Otsikko">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(242, 140, 40,0.8);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushHistoria">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>159</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushHistoria{
background-color:rgba(242, 140, 40,0.9); color: rgba(0, 0, 0, 0.8); border-style: outset;  border-width: 2px;   border-color: rgba(255, 140, 40,1);
color: white;
}
QPushButton#pushHistoria:hover{
background-color:rgba(230, 120, 40,0.9); color: rgba(0, 0, 0, 0.8); 
}</string>
   </property>
   <property name="text">
    <string>Historia</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_saldo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(242, 140, 40,0.8);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
